docs/src/pages/[platform]/theming/style-attributes/index.page.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Component Style Attributes
+metaDescription: 'CSS Style Attributes that are available on every amplify-ui component'
+supportedFrameworks: react
+---
+
+import { Example, ExampleCode } from '@/components/Example';
+import { ComponentPropsToStylePropsMap, Card } from '@aws-amplify/ui-react';
+import {
+  BackgroundStyles,
+  BorderStyles,
+  ColorStyles,
+  DesignTokenTable,
+  FlexStyles,
+  GridStyles,
+  LayoutStyles,
+  MarginAndPaddingStyles,
+  PositionStyles,
+  ShadowStyles,
+  SizeStyles,
+  TypographyStyles,
+  OtherStyles,
+} from './examples';
+
+These style attributes are available on amplify-ui primitive components and can be used to modify the corresponding styles. This allows users to quickly make style adjustments without having to go through css or add a theme object.
+
+### Background Styles
+
+<ExampleCode>
+
+```tsx
+import { Card, useTheme } from '@aws-amplify/ui-react';
+
+//use any CSS value
+<Card backgroundColor="red">Background Styling Example</Card>;
+
+//use a design token from the theme object
+const { tokens } = useTheme();
+<Card backgroundColor={tokens.colors.brand.primary[40]}>
+  Background Styling Example
+</Card>;
+
+//use a design token name
+<Card backgroundColor="brand.primary.40">Background Styling Example</Card>;
+```
+
+</ExampleCode>
+
+<BackgroundStyles />
+
+### Border Styles
+
+```tsx
```

Comment:
Can we make all of the example code copy/pastable? They’ll need to be refactored to actual functions that would render when copy/pasted. 

Also, can we also have visual demos of each example using our `Example` and `ExampleCode` components?

Reply:
Examples have been updated and are copy/pastable